### PR TITLE
Test against multiple Rails versions and Ruby 2.3

### PR DESCRIPTION
**Pull request by shaneog**
Use [Appraisal](https://github.com/thoughtbot/appraisal) to test against multiple Rails versions

Also:
- fix tests to not fail on various Rails 4 versions _(the order of the image tag attributes changed between Rails 4.1 and 4.2)_
- test against Ruby 2.3
- since the gem only works on Rails 4+ I added a runtime dependency for this
- fix tests to use correct domain name
- removed _Gemfile.lock_ since the recommendation is not to include this for gems

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,5 @@ config/secrets.yml
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
 *.gem
+Gemfile.lock
+*.gemfile.lock
```
Comment (rafaelcaricio) [inline .gitignore]: Could you please add a new-line here?

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,22 @@ language: ruby
...
+script: bundle exec rake
+matrix:
+  fast_finish: true
+  exclude:
+    - gemfile: gemfiles/rails_5.0.gemfile
+      rvm: 2.0
+    - gemfile: gemfiles/rails_5.0.gemfile
+      rvm: 2.1
+    - gemfile: gemfiles/rails_5.0.gemfile
+      rvm: 2.2
```
Comment (rafaelcaricio) [inline .travis.yml]: New line missing in the end of file.

File: Appraisals [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  gem 'rails', '~> 4.1.0'
+end
+
+appraise 'rails-4.2' do
+  gem 'rails', '~> 4.2.0'
+end
+
+appraise 'rails-5.0' do
+  gem 'rails', '>= 5.0.0.alpha', '< 5.1'
+end
```
Comment (rafaelcaricio) [inline Appraisals]: Again.

**Comment by rafaelcaricio** [comment]
@shaneog Thank you very much for your PR. :heart_decoration:  I really appreciate the changes you've made. Please just add those new-lines in the end of files and I will be glad to merge. Thank you!

**Comment by shaneog** [comment]
@rafaelcaricio I've made those changes. 
I also added a `.editorconfig` file so that things like trailing newlines are standardised for anyone making changes :smile:

**Comment by rafaelcaricio** [comment]
:+1: Done.

**Pull request merged**